docker/chainnode/entrypoint.sh

```diff
@@ -81,26 +81,40 @@ accountsCount=$(
 )
 
 # private key
-if [[ $accountsCount -le 0 ]]; then
-  echo "No accounts found"
-  if [[ ! -z $PRIVATE_KEY ]]; then
+if [[ ! -z $PRIVATE_KEY ]]; then
+  echo "$PRIVATE_KEY" > ./private_key
+  if [[ $accountsCount -le 0 ]]; then
+    echo "No accounts found"
     echo "Creating account from private key"
```

Comment:
Should we check mine flag here ?

Reply:
This is the import account part which is before the actual run of ronin node, so I don't think checking mine flag is needed here